app/controllers/landing_page_controller.rb

```diff
@@ -1,5 +1,12 @@
 class LandingPageController < ApplicationController
   def show
     @form = SearchForm.new
+    @remote_form = RemoteSearchForm.new
+  end
+
+  private
+
+  def page
```

Comment:
I think we can pull this up to the `ApplicationController` now it's shared across both types of search plus given pagination is going to be fairly pervasive throughout the rest of the application.